flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/SqlToOperationConverter.java

```diff
@@ -547,6 +547,12 @@ private Operation convertAlterTableOptions(
             ObjectIdentifier tableIdentifier,
             CatalogTable oldTable,
             SqlAlterTableOptions alterTableOptions) {
+        Map<String, String> setOptions =
```

Comment:
Will it be better to do the validation when parse `alter table set` in `parserImpls.ftl`
For me,  it looks like a restriction in sql synax level.  From my sense, `alter table cat1.db1.tb1 set ()` is like a illegal sql.
Anyway, looks good to me for  either way .

Reply:
I think we can follow the `ALTER TABLE ... RESET ...` which check empty also in here.